nixos/modules/security/acme.nix

```diff
@@ -173,6 +189,15 @@ in
         '';
       };
 
+      acceptTerms = mkOption {
```

Comment:
What do other people think of this? Personally I think it is overkill... but am fairly indifferent I guess :man_shrugging:

Reply:
If Let's Encrypt wants users to accept their terms I don't think implicitly accepting it for endusers should be the way to go. This knob is the exact place where we can point users to the terms of services of Lets Encrypt.

This doesn't make much sense though when overriding the API endpoints (`-s|--server`) because then we can't know where the correct ToS are.

Reply:
I don't have a really strong feeling about this. We  were implicitly accepting it before no? This seems to make the behaviour _slightly_ better than before